cli/src/config.rs

```diff
@@ -222,4 +223,19 @@ mod tests {
         let config: Config = parse_config(&test_config_file).unwrap();
         assert_eq!(config.request_type, PackageType::Npm);
     }
+
+    #[test]
+    fn test_pass_api_key_through_env() {
+        write_test_config();
+        let temp_dir = temp_dir();
+        let test_config_file = temp_dir.as_path().join("test_config");
+        env::set_var("PHYLUM_API_KEY", "ENV VARIABLE TOKEN");
+
+        let config: Config = read_configuration(&test_config_file).unwrap();
+
+        assert_eq!(
+            String::from(&config.auth_info.offline_access.unwrap()),
```

Comment:
This was the most immediately obvious way of doing string comparison on the `RefreshToken` type. If there is a better / more idiomatic way of doing this let me know!

Reply:
I don't think there's a better way to do this without touching `phylum-types`. Ideally that would have `PartialEq` derived, but unfortunately it does not.

Though you can skip the `String::from` for the token:

```rust
        assert_eq!(
            String::from(&config.auth_info.offline_access.unwrap()),
            token
        );
```

Reply:
The latest version of `phylum-types` now does implement `PartialEq`, so it should be possible to clean this up even further.